docs/quickstart.rst

```diff
@@ -185,24 +166,22 @@ A SyncroSim `Library`_ is a file (with *.ssim* extension) that stores all of you
     # Create a new Library
```

Comment:
Add second option for creating a new library where you don't specify a package when creating the library. Instead, you create an empty library, then add a package using `my_library.add_packages("helloworldTimePy")`

Reply:
my_library.**install**_packages("helloworldTimePy)?